tbx/static_src/sass/components/_division-signpost.scss

```diff
@@ -0,0 +1,81 @@
+@use 'config' as *;
+
+.division-signpost {
+    margin-bottom: $spacer-large;
+
+    &__intro {
+        margin-bottom: $spacer-medium;
+    }
+
+    &__cards {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: column;
+        gap: $spacer-mini-plus;
+
+        @include media-query(medium) {
+            flex-direction: row;
+        }
+    }
+
+    &__card {
+        flex-basis: 100%;
+        display: flex;
+        flex-direction: column;
+        transition: background-color $transition-quick;
+
+        @include media-query(medium) {
+            flex-basis: calc(50% - #{$spacer-mini-plus} * 0.5);
+        }
+
+        @include media-query(large) {
+            flex-basis: calc(33.33% - ($spacer-mini-plus * 2 / 3));
+        }
+
+        @include high-contrast-mode() {
+            border: 1px solid var(--color--text);
+        }
+
+        &--theme-coral {
+            background-color: var(--color--coral-charity);
+
+            &:has(.button-link:hover) {
```

Comment:
Praise: This is one of my favourite things to come to CSS recently! So powerful : )